src/js/core/widget/core/Appbar.js

```diff
@@ -440,7 +441,7 @@
 				 * so Page should be refreshed to better fit
 				 * to new available space.
 				 */
-				ns.engine.getBinding(this._ui.page, "Page").refresh();
+				utilsEvents.trigger(this.element, "appbartransitionend");
```

Comment:
Where is defined "appbartransitionend"?
I can't find it.

Reply:
This is signal to developer that AppBar has changed own size. Developer can resize height of content if it is needed. I don't want to force page refresh because this is not always required.